go.mod

```diff
@@ -36,9 +36,9 @@ require (
 	github.com/m4ksio/wal v1.0.0
 	github.com/mitchellh/mapstructure v1.3.3 // indirect
 	github.com/multiformats/go-multiaddr v0.3.1
-	github.com/onflow/cadence v0.18.1-0.20210624193306-81999eb7875b
-	github.com/onflow/flow-core-contracts/lib/go/contracts v0.7.3-0.20210602203947-82cffc9c55ed
-	github.com/onflow/flow-core-contracts/lib/go/templates v0.7.3-0.20210602203947-82cffc9c55ed
+	github.com/onflow/cadence v0.18.1-0.20210621164449-25b6135332eb
+	github.com/onflow/flow-core-contracts/lib/go/contracts v0.7.3-0.20210708181502-81b75f1a0ff8
+	github.com/onflow/flow-core-contracts/lib/go/templates v0.7.3-0.20210708181502-81b75f1a0ff8
```

Comment:
These have just been merged to `master` (🎉 ) so we should be able to pin to `v0.7.4` ([release](https://github.com/onflow/flow-core-contracts/releases/tag/lib%2Fgo%2Ftemplates%2Fv0.7.4))